packages/api/src/resolvers/function_resolvers.ts

```diff
@@ -319,6 +319,19 @@ export const functionResolvers = {
       if (article.wordCount) return article.wordCount
       return article.content ? wordsCount(article.content) : undefined
     },
+    async labels(
```

Comment:
How come this change is needed?

Reply:
hah, the labels we got from `inner join` do not have `source` in it which i think is something with typeorm 😭